### ISSUE TITLE
investigate documentation tools, choices and structure

### ISSUE THREAD

**Issue by jjgomezcadenas** [next-exp/IC#46]
We need to add a /docs directory to the repository. Proposal is to add it at the top level (next to invisible_cities), possibly with some extra sub-structure (such as /cities, /analysis, /core, etc.)

Open questions:
1) JJ argues that adding clean, well structured jupyter nb as docs is OK and good.  They need to be reviewed anyway, before entering repository.
2) Are we using Sphinx for docs? Any other tool?
3) What would be the structure of the /docs recommended by Sphinx?
4) Any other issues, suggestions?

**Comment by jjgomezcadenas** [comment]
Further thinking leads to the proposal that we should keep an independent NB repository (already available in nextic) for NB docs. Documentation with Sphinx or similar tools can go into the IC repository

**Comment by jmbenlloch** [comment]
I have tested sphinx in my fork. The idea is to have a doc/ folder containing one conf.py file for sphinx and as many rst files as we need for our documentation. This can also be linked to readthedocs.io, so we have a web like this http://ic-1.readthedocs.io/en/latest/ The documentation is recompiled for each commit automatically (we can also choose to have some "stable" versions).

I have made a pull request with the required changes in #84
